### PR TITLE
feat: Healthcheck endpoint

### PR DESCRIPTION
**Pull request by godexsoft**
This implement a simple endpoint for system healthcheck.

### CODE REVIEW

File: src/etlng/impl/GrpcSource.cpp [hunk 1]
```diff
@@ -49,10 +48,8 @@ resolve(std::string const& ip, std::string const& port)
 {
     web::Resolver resolver;
 
-    if (auto const results = resolver.resolve(ip, port); not results.empty()) {
-        std::cout << "resolved ip: '" << results.at(0) << '\n';
```
Comment (godexsoft) [inline src/etlng/impl/GrpcSource.cpp:53]: Oops :)

File: src/app/ClioApplication.cpp [hunk 2]
```diff
@@ -178,6 +186,16 @@ ClioApplication::run(bool const useNgWebServer)
             }
         );
 
+        httpServer->onGet(
```
Comment (godexsoft) [inline src/app/ClioApplication.cpp:189]: This will be a nuisance when merging back into `release/2.3.0`

File: src/app/ClioApplication.cpp [hunk 3]
```diff
@@ -178,6 +186,16 @@ ClioApplication::run(bool const useNgWebServer)
...
+            "/health",
```
Comment (kuznetsss) [inline src/app/ClioApplication.cpp:190]: Maybe `/ping`? Or `/healthcheck`?

File: tests/common/util/TestHttpClient.cpp [hunk 4]
```diff
@@ -58,7 +59,7 @@ using tcp = boost::asio::ip::tcp;
 
 namespace {
 
-std::string
+std::pair<boost::beast::http::status, std::string>
```
Comment (kuznetsss) [inline tests/common/util/TestHttpClient.cpp:62]: Maybe a structure would be better?

File: src/app/ClioApplication.cpp (hunk 3 above)
Comment (cindyyan317) [reply src/app/ClioApplication.cpp:190]: Maybe /ping will be confusing. Because we have ping API

File: src/etlng/impl/GrpcSource.cpp (hunk 1 above)
Comment (cindyyan317) [reply src/etlng/impl/GrpcSource.cpp:53]: 😨

**Review by cindyyan317 — APPROVED** [review]

File: src/app/ClioApplication.cpp (hunk 3 above)
Comment (godexsoft) [reply src/app/ClioApplication.cpp:190]: `/health` is fine

File: tests/common/util/TestHttpClient.cpp (hunk 4 above)
Comment (godexsoft) [reply tests/common/util/TestHttpClient.cpp:62]: That's what `pair` is for. If it had 3 or more members then yeah.

Comment (kuznetsss) [reply tests/common/util/TestHttpClient.cpp:62]: There is `tuple` for 3 and more 🙂 . I would say `pair` is bad for readability in general. But I'm ok if you don't want to change this part.

**Review by kuznetsss — APPROVED** [review]

**Pull request merged**